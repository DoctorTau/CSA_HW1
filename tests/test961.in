382
8135 5679 1562 -651 -2963 -5529 -6773 8951 9209 -1469 -5682 -3140 2434 4898 1482 -8585 -3297 2705 8855 8026 -1896 8092 -9012 -7681 8979 6179 95 -2839 -675 582 4560 -2085 9444 3187 5546 -8366 8337 4371 9320 -6600 -1766 -7330 -2770 -5627 4778 2774 -4384 3544 5064 440 4343 3558 9606 -3873 9828 4985 679 -7819 8516 -469 -4016 8403 8910 -165 -2234 -8919 -8000 -1001 -3856 8763 -5004 -4641 7793 -4814 -7952 6224 -2024 -9428 -3621 -7429 1720 8843 -1866 320 -2314 -7768 -260 -3972 -18 -5476 2885 1174 -6740 -6174 1274 9894 -4546 4219 272 -6840 -6407 -2267 -9614 5418 7384 -1862 329 3623 -2533 9709 9916 -9349 -609 -7838 -5987 -6048 6328 -2008 4964 8177 440 2492 1830 4539 -1958 7347 -503 -7383 -1106 7443 -5522 -6188 -5182 1403 2982 8387 156 3857 8893 -7522 -4079 4917 -4381 8121 8836 -6298 -7952 660 -7699 -7687 -7636 -4532 4376 4245 9340 -2750 9551 -4172 9071 8072 -669 9031 -3605 1458 -2436 -4049 2123 -2646 -6642 7597 993 5754 9686 -5879 422 9762 7967 -327 5332 -4215 4480 -4599 -358 5655 -628 -5157 -2524 711 -3109 -9702 -6309 8693 4273 -1855 -2427 -384 -5243 4642 -9036 -4745 4317 -349 9314 7376 -1857 7736 -1764 2004 9050 189 3831 -8673 3464 917 -5041 5646 9142 -7614 -2932 6568 7828 9707 -2815 -1431 177 -5483 -1113 6426 7058 1386 -3572 3233 5452 8067 9039 324 2139 -124 3283 -6392 1379 -2547 -8021 7283 1764 195 2148 514 253 1782 -5638 -2763 9866 -2000 5895 -1118 -3947 8487 -2203 -2186 1759 5952 1717 9445 -74 5445 -6660 -7229 5628 6674 4776 -8212 -6829 -5585 5358 -1219 7690 -1855 3338 2523 -8547 -2268 -7031 -117 -58 -8124 -8133 -4747 6798 8226 -8266 -4127 7176 836 2962 435 -2838 3231 -5336 4015 -4008 4981 -3328 720 -7417 6164 -6965 -1883 -7213 -352 -5986 -6638 -3551 -210 5013 -5521 -1055 -9724 2088 7813 -6484 -3221 -6921 7095 6289 -6731 5928 -1281 -5722 -892 -275 9938 4544 -8807 -4389 -4054 7774 -2427 -4154 5730 -1368 9563 3438 -6039 2681 4763 3882 -2202 8467 -5719 -4704 -358 -9408 -7288 -8223 7217 -7437 6399 -4684 8319 -3719 -2412 5139 507 3883 7616 -3250 -7556 5677 -3774 -7348 -3367 5469 9735 5763 -4710 -7594 -5389 -2977 -3065 436 8689 