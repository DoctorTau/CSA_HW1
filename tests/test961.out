-5682 -3140 -1896 -9012 -8366 -6600 -1766 -7330 -2770 -4384 -4016 -2234 -8000 -3856 -5004 -4814 -7952 -2024 -9428 -1866 -2314 -7768 -260 -3972 -18 -5476 -6740 -6174 -4546 -6840 -9614 -1862 -7838 -6048 -2008 -1958 -1106 -5522 -6188 -5182 -7522 -6298 -7952 -7636 -4532 -2750 -4172 -2436 -2646 -6642 -358 -628 -2524 -9702 -384 -9036 -1764 -7614 -2932 -3572 -124 -6392 -5638 -2000 -1118 -2186 -74 -6660 -8212 -2268 -58 -8124 -8266 -2838 -5336 -4008 -3328 -352 -5986 -6638 -210 -9724 -6484 -5722 -892 -4054 -4154 -1368 -2202 -4704 -358 -9408 -7288 -4684 -2412 -3250 -7556 -3774 -7348 -4710 -7594 